ksy_schema.json

```
@@ -99,16 +99,23 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "id": { "$ref": "#/definitions/Identifier" },
-        "title": { "type": "string" },
+        "id": {
+          "$ref": "#/definitions/Identifier",
+          "description": "unique string that identifies this format. It should be identical to the file name without the `.ksy` extension, for example 'exe' for 'exe.ksy'. This name is used to infer name of the top-level type when generate the parser\n\nThis field is mandatory on top-level definition and shouldn't be used at nested levels"
+        },
+        "title": {
+          "type": "string",
+          "description": "brief name of this format, for example 'Windows PE executable'"
+        },
         "application": {
           "anyOf": [
             { "type": "string" },
             {
               "type": "array",
               "items": { "type": "string" }
             }
-          ]
+          ],
+          "description": "name(s) of application(s) that uses this format, in free form"
```

```suggestion
          "description": "name(s) of application(s) that uses this format"
```

IMHO _in free form_ brings no useful information, it seems obvious and unimportant to me.